### DISCUSSION TITLE
What properties to use for Custom layout for PNP-Filters

### DISCUSSION THREAD

**Discussion by shridha21** [microsoft-search/pnp-modern-search#1361] [Q&A]
Version: 4.3.0

Hi,
I am trying to use the custom layout for PNP-Filters, and was looking to use the Pnp-panel component. Since the properties used in the Pnp-panel componet are not defined in custom layout, they are only available in Panel layout. What values can use instead of {{@root.properties.layoutProperties.isLightDismiss}}, {{@root.properties.layoutProperties.isBlocking}}, {{@root.properties.layoutProperties.size}}

 <pnp-panel   data-is-open="{{@root.selectedOnce}}" 
                        data-is-light-dismiss="{{@root.properties.layoutProperties.isLightDismiss}}"
                        data-is-blocking="{{@root.properties.layoutProperties.isBlocking}}"
                        data-size="{{@root.properties.layoutProperties.size}}"
                        data-theme-variant="{{JSONstringify @root.theme}}"
                        data-panel-header-text="{{@root.properties.layoutProperties.headerText}}">

Thanks,
Shirdha

**Comment by FranckyC** [comment]
Hi @shridha21, not sure what you are trying to do. Can you explain your requirement a little but more?
